Public/Src/FrontEnd/MsBuild/PipConstructor.cs

```diff
@@ -455,6 +455,11 @@ private void SetProcessEnvironmentVariables(IReadOnlyDictionary<string, string>
             processBuilder.AddOutputFile(logDirectory.Combine(PathTable, "msbuild.err"), FileExistence.Optional);
             processBuilder.AddOutputFile(logDirectory.Combine(PathTable, "msbuild.prf"), FileExistence.Optional);
 
+            if (m_resolverSettings?.EnableBinLogTracing == true)
+            {
+                processBuilder.AddOutputFile(logDirectory.Combine(PathTable, "msbuild.binlog"), FileExistence.Optional);

```

Comment:
>"msbuild.binlog" [](start = 77, length = 16)

Inconsequential. It is always better to declare a file if we know it is going to be (uniquely) produced. Without this, the containing shared opaque takes ownership.